exchange/bittrex/signer.go

```
@@ -20,7 +21,9 @@ func (self Signer) GetKey() string {
 
 func (self Signer) Sign(msg string) string {
 	mac := hmac.New(sha512.New, []byte(self.Secret))
-	mac.Write([]byte(msg))
+	if _, err := mac.Write([]byte(msg)); err != nil {
+		log.Printf("Encode message error: %s", err.Error())
```

this function should return an error?